python/lsst/daf/base/propertyContainer/propertyContainerContinued.py

```diff
@@ -340,27 +339,30 @@ class PropertySet:
                  None: "Undef",
                  }
 
-    def get(self, name):
-        """Return an item as a scalar or array
+    def get(self, name, default=None):
+        """Return an item as a scalar, else default.
 
-        Return an array if the item is of numeric or string type and has
-        more than one value, otherwise return a scalar.
-
-        .. deprecated:: 20180-06
-                  `get` is superseded by `getArray` or `getScalar`
+        Identical to `getScalar` except that a default value is returned
+        if the requested key is not present.
 
         Parameters
         ----------
         name : ``str``
             Name of item
+        default : `object`, optional
+            Default value to use if the named item is not present.
 
-        Raises
-        ------
-        KeyError
-            If the item does not exist.
+        Returns
+        -------
+        value : any type supported by container
+            Single value of any type supported by the container, else the
```

Comment:
Please be explicit about which value is returned if there is more than one. It returns the last value in the list (which is the most recent value added if add was called repeatedly).

I found 4 instances that could use this improvement: get and getScalar for PropertySet and PropertyList